src/unix/android-ifaddrs.c

```diff
@@ -476,7 +476,6 @@ static int interpretAddr(struct nlmsghdr *p_hdr, struct ifaddrs **p_resultList,
                     l_addrSize += NLMSG_ALIGN(calcAddrLen(l_info->ifa_family, l_rtaDataSize));
                     l_addedNetmask = 1;
                 }
-		break;
```

Comment:
Can you insert a `/* Fall through. */` comment here to avoid future confusion? Thanks.

Also, ping @devnexen.

Reply:
Would it not be clearer to keep the `break` and explicitly `l_addrSize += NLMSG_ALIGN(calcAddrLen(l_info->ifa_family, l_rtaDataSize));` for the second address?

Reply:
Updated for the suggestion of @richardlau